nixos/modules/services/torrent/flood.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, utils, ... }:
+
+let
+  cfg = config.services.flood;
+in
+{
+  meta.maintainers = with lib.maintainers; [ thiagokokada ];
+
+  options.services.flood = {
+    enable = lib.mkEnableOption "flood";
+    package = lib.mkPackageOption pkgs "flood" { };
+    openFirewall = lib.mkEnableOption "" // {
+      description = "Whether to automatically open the necessary ports in the firewall.";
+      default = true;
+    };
```

Comment:
It is convention that `openFirewall` options default to false except in the case of sshd.

Reply:
Done, also added examples for `port` and `host` and updated the description in `openFirewall` to be more informative.